Assets/EnemyBulletProperties.asset

```diff
@@ -0,0 +1,18 @@
+%YAML 1.1
```

Comment:
Can you place this in the correct folder

Reply:
Can do! I moved it to _Weapons > Scripts > ScriptableObjects > Bullets